lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -850,6 +861,13 @@ public static function get_resolved_theme_uris( $theme_json ) {
 			return $resolved_theme_uris;
 		}
 
+		$theme_json_data               = $theme_json->get_raw_data();
+		$resolved_theme_uris_cache_key = md5( wp_json_encode( $theme_json_data ) );
+
+		if ( ! empty( static::$resolved_theme_uris_cache[ $resolved_theme_uris_cache_key ] ) ) {
+			return static::$resolved_theme_uris_cache[ $resolved_theme_uris_cache_key ];
+		}
+
 		$theme_json_data = $theme_json->get_raw_data();
```

Comment:
```suggestion
```

Remove duplicate call

Reply:
How did that get there? 😇 

Good spotting!